atlasdb-workload-server/src/main/java/com/palantir/atlasdb/workload/WorkloadCell.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.workload;
+
+public interface WorkloadCell {
+    Integer column();
```

Comment:
Slight deviation as to what I proposed earlier: we will have a single dynamic column, and then a value (previously I had two dynamic columns, and a value). My assumption here is for indexing we can just do a modulo to figure out the bucket.